### PR TITLE
J# 48665

### PR DESCRIPTION
**Pull request by Khalid-Shahin**
EvidenceVariable with research study data dictionary elements added

### CODE REVIEW

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 1]
```diff
@@ -568,5 +568,207 @@
...
+			<path value="EvidenceVariable.classifier"/>
+			<short value="Classification"/>
+			<definition value="Classification of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.datatype">
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml]: The ticket indicates a dataStorage element should be added and that dataType and path should be sub-elements of dataStorage?

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 2]
```diff
@@ -568,5 +568,207 @@
...
+		<element id="EvidenceVariable.classifier">
+			<path value="EvidenceVariable.classifier"/>
+			<short value="Classification"/>
+			<definition value="Classification of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml:607]: The ticket indicates classifier should have an Example binding?

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 3]
```diff
@@ -568,5 +568,207 @@
...
+		<element id="EvidenceVariable.datatype">
+			<path value="EvidenceVariable.datatype"/>
+			<short value="Type of data used to express value of the variable"/>
+			<definition value="The type of data used to express values of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml:638]: The ticket indicates the dataStorage.dataType element should have a Preferred binding, but I don't see a binding?

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 4]
```diff
@@ -568,5 +568,207 @@
...
+		<element id="EvidenceVariable.constraint.conditional">
+			<path value="EvidenceVariable.constraint.conditional"/>
+			<short value="When this constraint applies"/>
+			<definition value="When this constraint applies."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml:685]: The ticket indicates contraint.conditional should have an Example binding?

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 5]
```diff
@@ -568,5 +568,207 @@
...
+		<element id="EvidenceVariable.constraint.code">
+			<path value="EvidenceVariable.constraint.code"/>
+			<short value="Rule for acceptable data values"/>
+			<definition value="A rule, such as a format or other expectation, for the data values."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml:762]: The ticket indicates code should have an Example binding

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 6]
```diff
@@ -568,5 +568,207 @@
...
+		<element id="EvidenceVariable.missingDataMeaning">
+			<path value="EvidenceVariable.missingDataMeaning"/>
+			<short value="How missing data can be interpreted"/>
+			<definition value="A method or transformation applied for missing data."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml:821]: The ticket indicates missingDataMeaning should have an example value set?

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml [hunk 7]
```diff
@@ -568,5 +568,207 @@
...
+		<element id="EvidenceVariable.unacceptableDataHandling">
+			<path value="EvidenceVariable.unacceptableDataHandling"/>
+			<short value="How erroneous data is processed"/>
+			<definition value="A method or transformation applied for data that does not match required patterns."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```
Comment (brynrhodes) [inline source/evidencevariable/structuredefinition-EvidenceVariable.xml:838]: The ticket indicates unacceptableDataHandling should have an example value set?

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Several changes requested in the comments based on the resolved disposition in the ticket. Also, can we add some examples illustrating the use of the new elements?

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 1 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml]: You're correct. Fixed.

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 2 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml:607]: Added the binding.

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 3 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml:638]: Added the binding.

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 4 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml:685]: Added the binding.

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 5 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml:762]: Added the binding.

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 6 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml:821]: Added the binding.

File: source/evidencevariable/structuredefinition-EvidenceVariable.xml (hunk 7 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/structuredefinition-EvidenceVariable.xml:838]: Added the binding.

File: source/evidencevariable/valueset-evidence-variable-unacceptable-data-handling.xml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cds"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="5"/>
```
Comment (brynrhodes) [inline source/evidencevariable/valueset-evidence-variable-unacceptable-data-handling.xml]: This is a new value set, fmm should be 0 or 1

File: source/evidencevariable/valueset-evidence-variable-constraint.xml [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cds"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="5"/>
```
Comment (brynrhodes) [inline source/evidencevariable/valueset-evidence-variable-constraint.xml]: fmm should be 0 or 1

File: source/evidencevariable/valueset-evidence-variable-classifier.xml [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cds"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="5"/>
```
Comment (brynrhodes) [inline source/evidencevariable/valueset-evidence-variable-classifier.xml]: fmm should be 0 or 1

File: source/evidencevariable/codesystem-evidence-variable-unacceptable-data-handling.xml [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cds"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="5"/>
```
Comment (brynrhodes) [inline source/evidencevariable/codesystem-evidence-variable-unacceptable-data-handling.xml]: fmm should be 0 or 1

File: source/evidencevariable/codesystem-evidence-variable-constraint.xml [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cds"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="5"/>
```
Comment (brynrhodes) [inline source/evidencevariable/codesystem-evidence-variable-constraint.xml]: fmm should be 0 or 1

File: source/evidencevariable/codesystem-evidence-variable-classifier.xml [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cds"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="5"/>
```
Comment (brynrhodes) [inline source/evidencevariable/codesystem-evidence-variable-classifier.xml]: fmm should be 0 or 1

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Looks good other than the FMM's are 5 for the new code systems and value sets, I think that's just a copy/paste thing, those should probably be no more than 1

File: source/evidencevariable/valueset-evidence-variable-unacceptable-data-handling.xml (hunk 8 above)
Comment (Khalid-Shahin) [reply source/evidencevariable/valueset-evidence-variable-unacceptable-data-handling.xml]: I set it to 1 now.

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**